client/src/components/R&R/ReviewAddModal.jsx

```diff
@@ -1,15 +1,159 @@
+            <label htmlFor="recommend-no">No</label>
+          </div>
+          <div className="modal-characteristics" onChange={handleCharacteristic}>
+            <CharacteristicRadios chars={reviewsMeta.characteristics} />
+          </div>
+          <div className="modal-review-summary">
+            Review summary:
+            <input
+              className="review-text-input"
+              type="text"
+              maxLength="60"
+              placeholder="Example: Best purchase ever!"
+              onChange={(event) => {
+                setSummary(event.target.value);
+              }}
+            />
+          </div>
+          <div className="modal-review-body">
+            *Review body:
+            <input
+              required="required"
+              className="review-text-input"
+              type="text"
+              maxLength="1000"
+              placeholder="Why did you like the product or not?"
+              minLength="50"
+              onChange={(event) => {
+                setBody(event.target.value);
+              }}
+            />
+          </div>
+          <div>
+            {50 - body.length > 0 ? `*Minimum required characters left: ${50 - body.length}` : 'Minimum reached'}
+          </div>
+          <div className="modal-photos">
+            Upload photos:
+            <input
+              multiple="multiple"
+              type="file"
+              accept="image/png, image/jpeg"
+              onChange={handleImageUpload}
+            />
+            <div className="upload-thumbnails">
+              {images.length <= 5 ? images.map(
+                (image) => (
+                  <div className="upload-thumb-container">
+                    <img className="upload-preview" src={image} alt="uploadphoto" />
+                  </div>
+                ),
+              ) : <div className="max-photos">Maximum of 5 Photos allowed.</div>}
+            </div>
+          </div>
+          <div className="modal-nickname">
+            *Desired nickname:
+            <input
+              required="required"
+              className="review-text-input"
+              type="text"
+              maxLength="60"
+              placeholder="Example: jackson11!"
+              onChange={(event) => {
+                setNickname(event.target.value);
+              }}
+            />
+          </div>
+          <div className="modal-email">
+            *Your email?:
+            <input
+              required="required"
+              className="review-text-input"
+              type="email"
+              maxLength="60"
+              placeholder="Example: jackson11@email.com!"
+              onChange={(event) => {
+                setEmail(event.target.value);
+              }}
+            />
+          </div>
+          <div>For authentication reasons only, you will not be emailed</div>
```

Comment:
Good to include, but weirdly placed. I don't have any suggestions, I ain't a designer. They don't pay me enough.